RELEASE-NOTES.txt

```diff
@@ -1,5 +1,7 @@
 24.2
 -----
+* [**] Prevent images from temporarily disappearing when uploading media [https://github.com/WordPress/gutenberg/pull/57869]
+* [**] Fix crash occurring on large post on Android [https://github.com/WordPress/gutenberg/pull/58266]
```

Comment:
@derekblank Similar to https://github.com/wordpress-mobile/WordPress-Android/pull/20093#pullrequestreview-1858712761, we could add more detail to the release notes by adding the prefix `Block Editor: ` for entries related to the editor ([example](https://github.com/wordpress-mobile/WordPress-iOS/blob/8e8c5b41a7ae9d2c6a20d4bcf411506243554ee2/RELEASE-NOTES.txt#L13)).